api/admin/routes.py

```diff
@@ -401,15 +420,85 @@ def work_change_book_cover(identifier_type, identifier):
 def work_complaints(identifier_type, identifier):
     return app.manager.admin_work_controller.complaints(identifier_type, identifier)
 
+
 @library_route('/admin/works/<identifier_type>/<path:identifier>/lists', methods=['GET', 'POST'])
 @has_library
 @allows_cors(allowed_domain_type=set({"admin"}))
 @returns_json_or_response_or_problem_detail
 @requires_admin
 @requires_csrf_token
 def work_custom_lists(identifier_type, identifier):
+    """Returns or updates custom lists
+    ---
+    args:
+        identifier_type (str): A type of identifier, e.g. "ISBN"
+        identifier (str): An identifier string, used with `identifier_type` to look up an Identifier.
```

Comment:
These should be set as parameters in the `get` and `post` block so they show up in the the API docs. If it's useful they can be put into the controller and re-used for both. There should be a few examples available.